app/assets/javascripts/application.js

```diff
@@ -12,13 +12,8 @@
 //
 //= require jquery
 //= require jquery_ujs
-
-//= require materialize-sprockets
```

Comment:
@jklockri For reals? We don't need this?

Reply:
@jklockri Oh cool! I see. This does work, but only because someone added extra copes of `materialize.js` and `materialize.min.js` to our javascripts assets folder. I think the better way to get rid of those errors is to remove those files...and keep this line here since we have the gem loaded in our gem file.